index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
         </ul>
       </nav>
     </header>
+     
+    <!-- Mobile pop-up menu -->
+ 
+    <div class="mb_menu">
+      <img src="media/png/x.pn" alt="x icon" id="x_icon">
```

Comment:
- [x] Kindly consider updating the relative path for the image source as it doesn't work as expected.
```suggestion
      <img src="media/png/X.png" alt="x icon" id="x_icon">
```